_targets.R

```diff
@@ -5,27 +5,63 @@ source("3_visualize/src/plot_timeseries.R")
 
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
-
+#site numbers
+sites = c("01427207", "01432160","01435000", "01436690", "01466500")
+#parameter codes for query
+parameterCd = '00010' 
+startDate="2014-05-01" 
+endDate="2015-05-01"
+#parameters for plotting
 p_width <- 12
 p_height <- 7
 p_units <- "in"
 
 p1_targets_list <- list(
   tar_target(
-    site_data,
-    download_nwis_data(),
+    create_directories,
+    create_dirs(sites),
+    format = "file"
```

Comment:
See other comment about revising this step but here your function would return a list of file paths rather than create a file, so you wouldn't have the 'format = "file"'